source/documentation/dns/register_defensive_domain.html.md.erb

```diff
@@ -0,0 +1,79 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Register New Defensive Domain
+last_reviewed_on: 2024-10-16
+review_in: 6 months
+---
+
+# Register New Defensive Domain
+
+This document explains the process of registering a defensive domain at the Ministry of Justice.
+
+Security owns the policy for [Defensive Domain Registrations](https://security-guidance.service.justice.gov.uk/defensive-domain-registration/#defensive-domain-registrations). 
+
+Operations Engineering doesn't own the decisions around what should be a defensive domains, or what requires a defensive domains. We just ensure that the policy is implemented when a requirement is justified.
+
+## Pre-requisites
+
+- List of defensive domains that need to be registered
+- Approval from Security Team
```

Comment:
How do we get approval?

Reply:
I'll add some words.

Reply:
Perhaps add a "Before proceeding with the registration, ensure you have:" Just above this list.

So, like:
## Pre-requisites

Before proceeding with the registration, ensure you have:

- blah
- blah

Reply:
Wording update to clarify approval will be via email.